Doc/library/typing.rst

```diff
@@ -2029,6 +2029,22 @@ Functions and decorators
    runtime we intentionally don't check anything (we want this
    to be as fast as possible).
 
+.. function:: assert_type(val, typ, /)
+
+   Assert (to the type checker) that the value is of the given type.
```

Comment:
```suggestion
   Assert (to the type checker) that *value* has a revealed type of *typ*.
```

The type checker has no knowledge of the _actual type_ of `value`. It only has knowledge of its _inferred type_ of `value`, which is what this function tests.

Reply:
I don't think "revealed type" makes sense if we're not calling reveal_type():

```suggestion
   Assert (to the type checker) that *value* has an inferred type of *type*.
```